### PR TITLE
Add Circle component

### LINKED ISSUE #3348: New Sensor Flow/Page
## Is your feature request related to a problem? Please describe.

The new sensor flow and page

## Describe the solution you'd like

As per the marvel prototype

## Implementation

Quite a big page with a lot of steps. Going to break it down into the following tasks:

- [ ] `Stepper`
- [ ] Make `IconButton` have a `Rectangle` variant?
- [ ] Make step1 component
- [ ] Make step2 component
- [ ] Make step3 component
- [ ] Logic to send bluetooth commands + save a new sensor

## Describe alternatives you've considered

N/A

## Additional context

Resources

- https://marvelapp.com/prototype/55aea8f/screen/75648024
image

- https://drive.google.com/file/d/1wLg12JAQyrgB6D3F_X0O9LBZwJs8PcJM/view?usp=sharing
image

- https://drive.google.com/file/d/15SsY4YCmhpSgiihU1Q7FpZMnqCUkZcIp/view?usp=sharing
image

- https://drive.google.com/file/d/15SsY4YCmhpSgiihU1Q7FpZMnqCUkZcIp/view?usp=sharing
image

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #3348 

## Change summary

- Added a `Circle` component - was sick of dealing - used it in `CircleButton`
- Added a bit of a cheap hack to the stepper to choose between this new variant and the old one. I am hoping to change the old one to the new one.. 
- Added this stepper as an option to the WIZARD which "just works" and you can click on past steps and call the callback `switchTab` to change tabs
- Think this is sufficient enough for a PR even though there's not much real content yet

## Testing

Internal

### Related areas to think about

image

### CODE REVIEW

File: src/widgets/Circle.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/forbid-prop-types */
```
Comment (Chris-Petty) [inline src/widgets/Circle.js:1]: If we're doing this everywhere, should we just disable this in the linter config?

File: src/widgets/Circle.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  style: {},
+};
+
+Circle.propTypes = {
+  style: PropTypes.object,
+  size: PropTypes.number,
+  children: PropTypes.node,
+  backgroundColor: PropTypes.string,
+  borderColor: PropTypes.string,
+  elevate: PropTypes.number,
```
Comment (Chris-Petty) [inline src/widgets/Circle.js:45]: Boolean? Got error for this in console

**Review by Chris-Petty — CHANGES_REQUESTED** [review]
Proptype nit, otherwise looks good:

image

Couldn't test onPress for going back though cause I can't get past step 1 in dispensary #3323

File: src/widgets/Circle.js (hunk 2 above)
Comment (joshxg) [reply src/widgets/Circle.js]: 73ac547

Thanks!

File: src/widgets/Circle.js (hunk 1 above)
Comment (joshxg) [reply src/widgets/Circle.js:1]: I've suggested it but been overruled in the past!

**Review by Chris-Petty — APPROVED** [review]
Looks great! I did get to test it further, clicking to go back a step works a treat.

**Pull request merged**